db/schema.rb

```diff
@@ -95,7 +96,7 @@
     t.string "name"
     t.string "email"
     t.string "verification_token"
-    t.boolean "verified", default: false
+    t.boolean "verified", default: true
```

Comment:
Is this intended? I'm not seeing a correlating migration

Reply:
I'm getting this occasionally when running `rake db:migrate` locally.  Weird - this would be a bad bug.

Reply:
Not intended, Nick do you ever see this in other branches too? I commented out my migrations and still saw this bit flip. 

We could fix this with the line 
```ruby
    change_column_default :legacy_publishers, :verified, false
```

I can put this in the ```RenameRecreatePublishers```, migration - where legacy publishers was created. Although I've been following Cory's advice not to modify older migrations. 

Alternatively, I can create a new migration for the above line, let me know which you guys would prefer.